Tests/MSFT_xIPAddress.Tests.ps1

```diff
@@ -25,6 +38,18 @@ InModuleScope MSFT_xIPAddress {
                 $Result.IPAddress | Should Be $Splat.IPAddress
             }
         }
+
+        Context 'Subnet Mask' {
+            It 'should fail if passed a negative number' {
+                $Splat = @{
+                    IPAddress = '192.168.0.1'
+                    InterfaceAlias = 'Ethernet'
+                    Subnet = -16
+                }
+
+                { Get-TargetResource @Splat } | Should throw
```

Comment:
Suggestion: you should verify the message, this could be throwing for any number of reasons.  [pester should throw reference](https://github.com/pester/Pester/wiki/Should#throw)